mongo_connector/doc_managers/neo4j_doc_manager.py

```diff
@@ -33,13 +31,14 @@ class DocManager(DocManagerBase):
   def __init__(self, url, auto_commit_interval=DEFAULT_COMMIT_INTERVAL,
                  unique_key='_id', chunk_size=DEFAULT_MAX_BULK, **kwargs):
     self.remote_graph = Graph(url, **kwargs.get('clientOptions', {}))
-    graph = Graph()
+    self.graph = Graph()
```

Comment:
Questions deriving from here:
- SHould we use Graph or remote graph?
- We will need to make the authentication step. So I'll add these parameters into the constructor also. ( http://py2neo.org/2.0/essentials.html#authentication )